orcid-web/src/main/webapp/static/javascript/script.js

```diff
@@ -129,24 +129,29 @@ var OrcidCookie = new function() {
     };
 };
 
-var _gaq = _gaq || [];
-
 var OrcidGA = function() {
     // test and make sure _gaq is working. disconnect.me chrome plugin has
     // caused silent _gaq failures. This check allows us to detect that
     // situation
-    var gaEnabled = false;
-    _gaq.push(function() {
-        gaEnabled = true;
-    });
+    var gaEnabled = true;
```

Comment:
Ok, am testing http://davidwalsh.name/detecting-google-analytics, I will let you know